test/functional/sqlsrv/sqlsrv_ae_azure_key_vault_username_password.phpt

```diff
@@ -0,0 +1,45 @@
+--TEST--
+Test username/password credentials for Azure Key Vault for Always Encrypted.
+--SKIPIF--
+<?php require('skipif_versions_old.inc'); ?>
+--FILE--
+<?php
+require_once('sqlsrv_ae_azure_key_vault_common.php');
+
+$strsize = 64;
+
```

Comment:
you use 64 in almost every AKV test -- why not create a constant in values.php and reference it in all your tests?

Reply:
That's a good idea...